src/main/kotlin/calculator/Operator.kt

```diff
@@ -0,0 +1,46 @@
+package calculator
+
+enum class Operator : MathElement {
+    Addition {
```

Comment:
```suggestion
enum class Operator(private val symbol: String) : MathElement {
    Addition("+") {
```
요렇게도 구현 가능할 것 같습니다 😄